Bio/Align/Applications/_Guidance2.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 by Rob Gilmore and Shaurita Hutchins. All rights reserved.
+# Based on ClustalOmega wrapper copyright 2011 by Andreas Wilm.
+#
+# Wrapper for Guidance2 by Rob Gilmore (2017).  http://guidance.tau.ac.il/ver2/
+# Used _ClustalOmega.py as template.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+"""Command line wrapper for the multiple alignment program, GUIDANCE2.
+It weights, filters or masks unreliably aligned positions in multiple sequence alignments.
+"""
+
+from __future__ import print_function
+
+import os
+from Bio.Application import _Option, AbstractCommandline
+
+
+class Guidance2Commandline(AbstractCommandline):
+    u"""Command line wrapper for GUIDANCE2.
+
+    http://guidance.tau.ac.il/ver2/
+
+    Example:
+    --------
+
+    >>> from Bio.Align.Applications import Guidance2Commandline
+    >>> import os
+    >>> seqFile = "ADGRB1.ffn"
+    >>> msaProgram = "CLUSTALW"
+    >>> seqType = "codon"
+    >>> outDir = Path(os.getcwd)
+    >>> Guidance2_cline = Guidance2Commandline(seqFile, msaProgram, seqType, str(outDir), bootstraps=20, seqCutoff=0.63,\
+                                    colCutoff=0.9, outOrder='as_input', dataset='ADGRB1')
+    >>>> print(Guidance2_cline)
+    perl guidance.pl --seqFile HTR1A.ffn --msaProgram CLUSTALW --seqType codon --outDir ~/Guidance2/data
+    --bootstraps 20 --seqCutoff 0.63 --colCutoff 0.9 --outOrder as_input --dataset ADGRB1
+
+
+    You would typically run the command line with Guidance2_cline() or via
+    the Python subprocess module, as described in the Biopython tutorial.
+
+    Citation:
```

Comment:
We've starting moving towards the numpydoc docstring standard (cross reference #906 and #1388), which means using underlines section names in a specific order. Citation becomes References:

```
References
----------
```

See the updated https://github.com/biopython/biopython/blob/master/Bio/Align/Applications/_ClustalOmega.py#L18 for guidance.

Reply:
Btw.  Super excited for sphinx documentation with restructured text.